admin/server/organizations.go

```diff
@@ -328,6 +328,15 @@ func (s *Server) RemoveOrganizationMember(ctx context.Context, req *adminv1.Remo
 	if err != nil {
 		return nil, status.Error(codes.Internal, err.Error())
 	}
+
+	// delete from projects if RemoveFromProjects flag is set
+	if req.KeepProjectRoles {
```

Comment:
if `KeepProjectRoles` is true then we are deleting the roles, logic should be flipped ?

Reply:
Yeah it should be `if !req.KeepProjectRoles` then delete, 

But @begelundmuller should we keep `keep-project-roles` as false by default? it means we will delete the roles if this flag is not set, it that fine?